couchbase/json-nodes.yml

```diff
@@ -0,0 +1,63 @@
+type: json
+data:
+  url: /pools/default
+  server: ${COUCHBASE_HOSTPORT}
+  basicHttpAuthUsername: ${COUCHBASE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${COUCHBASE_BASICAUTH_PASSWORD}
+
+observation:
+  - name: nodes
+    metricNamespace: couchbase
+    path: $.nodes.[?(@.hostname=${name})].interestingStats
+
+    metric:
+      - name: items.active
+        source: curr_items
+        type: long_gauge
+        label: active items
+        description: Number of active items in memory
+
+      - name: items.total
+        source: curr_items_tot
+        type: long_gauge
+        label: total items
+        description: Total number of items
+
+      - name: docs.size
+        source: couch_docs_data_size
+        type: long_gauge
+        label: data size
+        unit: bytes
+        description: Couch docs data size in bytes
```

Comment:
I think we can remove the unit in the description since we have it in the `unit` field. Applicable to all descriptions with the unit.

Reply:
OK. removed